src/components/Field/index.js

```diff
@@ -41,144 +45,139 @@ class Field extends Component {
 
     isComplete = () => {
         let state = this.state.cards.every(el => {
-            return el.isActive;
+            return el.removed;
         });
         if (state) {
-            this.setState({timer: 'pause', complete: true});
+            this.setState({timerControl: {isPaused: true}, isComplete: true});
         }
     };
 
     shouldRemoveActive = (element) => {
         let temp = this.state.cards.slice();
         let index = this.state.cards.indexOf(element);
         temp[index].shouldRemove = true;
-        index = this.state.cards.indexOf(this.state.active);
+        index = this.state.cards.indexOf(this.state.activeCard);
         temp[index].shouldRemove = true;
         this.setState({cards: temp, preventClick: true});
     };
 
     removeActive = () => {
-        let temp = this.state.cards.filter((el) => {
-            return !el.shouldRemove;
+        let temp = this.state.cards.map((el) => {
+            if (el.shouldRemove) {
+                el.removed = true;
```

Comment:
Do you need to use both `removed` and `shouldRemove`? Maybe you can leave just one of them